scripts/build/simulations.mk

```diff
@@ -18,51 +19,45 @@ test-sim-nondeterminism:
+	# 	-NumBlocks=100 -BlockSize=200 -Seed=99  -SigverifyTx=false
 
 test-sim-import-export:
-	# @echo "Running application import/export simulation. This may take several minutes..."
-	# @cd ${CURRENT_DIR}/simapp && go test -failfast -mod=readonly -timeout 20m -tags='sims' -run TestAppImportExport \
-	# 	-NumBlocks=50 -Period=5
+#	 @echo "Running application import/export simulation. This may take several minutes..."
+#	 @cd ${CURRENT_DIR}/simapp/v2 && go test -failfast -mod=readonly -timeout 20m -tags='sims' -run TestAppImportExport \
+	# 	-NumBlocks=50
 
 test-sim-after-import:
-	# @echo "Running application simulation-after-import. This may take several minutes..."
-	# @cd ${CURRENT_DIR}/simapp && go test -failfast -mod=readonly -timeout 30m -tags='sims' -run TestAppSimulationAfterImport \
-	# 	-NumBlocks=50 -Period=5
+	 @echo "Running application simulation-after-import. This may take several minutes..."
+	 @cd ${CURRENT_DIR}/simapp/v2 && go test -failfast -mod=readonly -timeout 30m -tags='sims' -run TestAppSimulationAfterImport \
+	 	-NumBlocks=50
 
 test-sim-custom-genesis-multi-seed:
 	# @echo "Running multi-seed custom genesis simulation..."
 	# @echo "By default, ${HOME}/.simapp/config/genesis.json will be used."
-	# @cd ${CURRENT_DIR}/simapp && go test -failfast -mod=readonly -timeout 30m -tags='sims' -run TestFullAppSimulation -Genesis=${HOME}/.simapp/config/genesis.json \
-	# 	-NumBlocks=400 -Period=5
+	# @cd ${CURRENT_DIR}/simapp/v2 && go test -failfast -mod=readonly -timeout 30m -tags='sims' -run TestFullAppSimulation -Genesis=${HOME}/.simapp/config/genesis.json \
+	# 	-NumBlocks=400
 
 test-sim-multi-seed-long:
-	# @echo "Running long multi-seed application simulation. This may take awhile!"
-	# @cd ${CURRENT_DIR}/simapp && go test -failfast -mod=readonly -timeout=2h -tags='sims' -run TestFullAppSimulation \
-	# 	-NumBlocks=150 -Period=50
-
-test-sim-multi-seed-short: test-v2-sim
-	# @echo "Running short multi-seed application simulation. This may take awhile!"
-	# @cd ${CURRENT_DIR}/simapp && go test -failfast -mod=readonly -timeout 30m -tags='sims' -run TestFullAppSimulation \
-	#	-NumBlocks=50 -Period=10 -FauxMerkle=true
+	 @echo "Running long multi-seed application simulation. This may take awhile!"
+	 @cd ${CURRENT_DIR}/simapp/v2 && go test -failfast -mod=readonly -timeout=2h -tags='sims' -run TestFullAppSimulation \
+		-NumBlocks=150
 
-.Phony: test-v2-sim
-test-v2-sim:
-	@echo "Running short multi-seed application simulation. This may take awhile!"
-	@cd ${CURRENT_DIR}/simapp/v2 && go test -failfast -mod=readonly -timeout 30m -tags='sims' -run TestSimsAppV2 \
-#		-NumBlocks=50 -Period=10 -FauxMerkle=true
+test-sim-multi-seed-short:
+	 @echo "Running short multi-seed application simulation. This may take awhile!"
+	 @cd ${CURRENT_DIR}/simapp/v2 && go test -failfast -mod=readonly -timeout 30m -tags='sims' -run TestFullAppSimulation \
+		-NumBlocks=50
 
 test-sim-benchmark-invariants:
```

Comment:
We can remove this as invariants have been removed

Reply:
👍 I will remove this when I touch the benchmarks